examples/nextjs-v15-app/next.config.js

```diff
@@ -0,0 +1,11 @@
+const withLitSSR = require('@lit-labs/nextjs')({
+  addDeclarativeShadowDomPolyfill: true,
+});
+
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  reactStrictMode: true,
+  swcMinify: true,
```

Comment:
I'm seeing the following when running so they must've removed the option in this version.
```
 ⚠ Invalid next.config.js options detected: 
 ⚠     Unrecognized key(s) in object: 'swcMinify'
 ⚠ See more info here: https://nextjs.org/docs/messages/invalid-next-config
```

```suggestion
```